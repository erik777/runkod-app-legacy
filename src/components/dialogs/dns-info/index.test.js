import React from 'react';
import {DialogContent} from './index';
import TestRenderer from 'react-test-renderer';


test('1- default', () => {
  const props = {
    toggleUiProp: () => {
    }
  };

  const renderer = TestRenderer.create(
    <DialogContent {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});

