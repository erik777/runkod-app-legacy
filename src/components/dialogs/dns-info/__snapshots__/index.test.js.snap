// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default 1`] = `
<div
  className="user-select"
>
  <p>
    First, go to the DNS settings page of your domain provider.
  </p>
  <p>
    Then, Create an A record: type @ for "Host" and paste 178.128.141.114 under "Value" or "Points to".
  </p>
  <img
    alt="Dns Settings"
    className="sc-img"
    src="dns-settings.png"
  />
  <p>
    If you want to redirect www subdomain add CNAME record like above.
  </p>
  <p
    className="text-muted"
  >
    Please keep in mind, depending on your dns provider it may take some time to activate new settings.
  </p>
</div>
`;
